torch/nested/_internal/ops.py

```diff
@@ -1678,6 +1678,31 @@ def embedding_default(func, *args, **kwargs):
     )
 
 
+@register_jagged_func(
+    torch.ops.aten.embedding_dense_backward.default,
+    "self: jt, grad_output: jt, num_weights: any, padding_idx: any, scale_grad_by_freq: any",
+)
+def embedding_dense_backward(func, *args, **kwargs):
+    _, new_kwargs = normalize_function(  # type: ignore[misc]
+        func, args=args, kwargs=kwargs, normalize_to_only_use_kwargs=True
+    )
+
+    indices: NestedTensor = new_kwargs.pop("indices")
+    num_weights: int = new_kwargs.pop("num_weights")
+    grad_output: NestedTensor = new_kwargs.pop("grad_output")
+    if new_kwargs["scale_grad_by_freq"]:
+        raise NotImplementedError
```

Comment:
Happy to try adding support for when `scale_grad_by_freq` is `True`...

Reply:
I think we'll get this for free by redispatching onto the underlying `func` :) (see my comment below)